hack/ci/create-pull-request.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```

Comment:
I don't think this kind of script makes sense for this repo. I assume you copied it from the kubelet repo?

Reply:
yep, deleted it